### DISCUSSION TITLE
Prometheus error

### DISCUSSION THREAD

**Discussion by CCTV-six** [prometheus/prometheus#14646] [Q&A]
### What did you do?

1.Set Prometheus and Targets
2.In the Prometheus configuration file, set the scrape_timeout field to 5s

### What did you expect to see?

It is expected that after addressing the root cause of delay or slow response, Prometheus will be able to successfully capture metrics for targets without any timeout errors

### What did you see instead? Under which circumstances?

level=error ts=2023-04-01T12:33:44.563Z caller=scrape.go:1377 component="scrape manager" scrape_pool=my-target err="context deadline exceeded"

**Comment by machine424** [comment]
Maybe the scrape attempts are still timing out.
You can stop prometheus and try to reach the target (exactly as Prometheus would do) in a loop, I bet those requests would take more than `5s` sometimes.
I'll turn this into a discussion as it seems as a support request for now.
